examples/active_qubit_reset.md

```diff
@@ -0,0 +1,518 @@
+    name: python3
+---
+
+# Active Qubit Reset
+
+
+This tutorial shows how to perform an active qubit reset experiment with real-time feedback using Zurich Instruments PQSC, SHFQA and SHFSG devices.
+
+Active qubit reset is a procedure to initialize the qubit in the ground state by measuring its state: if the qubit is measured in the exited state, then a $\pi$ pulse is applied to bring it in the ground state, while if the qubit is measured in the ground state then it's already initialized and there's nothing else to be done. In this example we will use the SHFQA to perform the readout of the qubit's state, the SHFSG to apply the conditional resetting pulse, and the PQSC to coordinate the experiment, enabling real-time feedback pulse with as little latency as possible.
+
+Requirements:
+- LabOne Version >= 22.08
+- 1 SHFQA
+- 1 SHFSG
+- 1 PQSC
+- ZSync connections between the PQSC and the SHFQA and between the PQSC and the SHFSG
+- Loopback connection between input and output of channel 0 of the SHFQA
+----
+
+```python
+from zhinst.toolkit import Session, SHFQAChannelMode, Waveforms, Sequence, CommandTable
+import pqsc_helpers
+import numpy as np
+import time, textwrap
+```
+
+## Connect to the devices
+
+
+Create a session with a running Data Server, and then connect the Data Server with the instruments.
+
+```python
+shfqa_serial = "DEVXXXX"
+shfsg_serial = "DEVYYYY"
+pqsc_serial = "DEVZZZZ"
+
+session = Session("localhost")
+
+shfqa = session.connect_device(shfqa_serial)
+shfsg = session.connect_device(shfsg_serial)
+pqsc = session.connect_device(pqsc_serial)
+```
+
+## Basic initial setup
+
+
+### Clocks and communications
+The PQSC can either receive the clock from an external source, as shown in this example, or generate it internally. Then, the PQSC distributes the clock to the other devices via ZSync connections.
+
+```python
+# PQSC external reference clock
+pqsc.system.clocks.referenceclock.in_.source("external")
+pqsc.check_ref_clock()
+
+# SHFSG and SHFQA ZSync clock
+shfsg.system.clocks.referenceclock.in_.source("zsync")
+shfqa.system.clocks.referenceclock.in_.source("zsync")
+```
+
+### Inputs and outputs
+
+
+Configure the basic settings for the SHFQA and SHFSG inputs and outputs, such as the range and the center frequency. For the PQSC we configure that the trigger signals sent over ZSync are replicated on the trigger output; the trigger output can then be connected to an oscilloscope to visualize the signals.
+
+```python
+# Default channels
+QA_CHANNEL = 1
+SG_CHANNEL = 0
+
+with session.set_transaction():
+    # SHFQA
+    shfqa.qachannels[QA_CHANNEL].configure_channel(
+        center_frequency=1e9,
+        input_range=0,
+        output_range=0.0,
+        mode=SHFQAChannelMode.READOUT,
+    )
+    shfqa.qachannels[QA_CHANNEL].input.on(True)
+    shfqa.qachannels[QA_CHANNEL].output.on(True)
+
+    # SHFSG
+    shfsg.sgchannels[SG_CHANNEL].awg.diozsyncswitch("zsync")
+    shfsg.sgchannels[SG_CHANNEL].output.on(True)
```

Comment:
Please move the `.on(True)` after the setting of frequency and range. Please add the units of frequency and range